src/plugins/esql_datagrid/public/row_viewer.test.tsx

```diff
@@ -106,14 +106,14 @@ describe('RowViewer', () => {
       },
     } as unknown as DataTableRecord);
     await waitFor(() => {
-      expect(screen.getByTestId('esqlTableRowNavigation')).toBeInTheDocument();
+      expect(screen.getByTestId('dscDocNavigation')).toBeInTheDocument();
```

Comment:
Can we make the test sunj here more generic and not discover specific? (As it is a shared component now)

Reply:
Updated in https://github.com/elastic/kibana/pull/175787/commits/dd680be08e7cf5ef7460a0545bfacefc4c607e6f